src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -811,6 +811,14 @@ public synchronized void close()
             // is removed.
             closed = true;
 
+            if (singlePortHarvesters != null) {
```

Comment:
This will close the harvesters which are shared for the whole jitsi-videobridge instance (effectively stopping the bridge) once a single transport manager instance is closed (e.g. one participant exits a conference).

Reply:
That definitely would be a bug (not a feature).